Servers/using-runner-to-configure-disks-on-bare-metal-servers.md

```diff
@@ -0,0 +1,114 @@
+
+    ![Bare Metal Storage Configuration](../images/bare_metal_runner_disks_2.png)
+
+    ![Run Bare Metal Storage Configuration](../images/bare_metal_runner_disks_3.png)
+
+4. Please pay special attention to the warning notice regarding Disk Settings and Configuration.
+
+  ![Bare Metal Storage Configuration](../images/bare_metal_runner_disks_4.png)
+
+5. **Datacenter Picker** - Select the data center that contains the Bare Metal server you would like to configure disks.
+6. **Servers (Add To Inventory)** - Select one or more Bare Metal servers, with a configurable disk array controller, in that data center.
+7. **Controller Personality** - Select the Controller Personality:
+  - RAID-Mode (RAID-x)
+  - HBA-Mode (JBOD)
+8. **Partition/Format/Mount All Raw Disks** - Select whether you would like Runner to create a partition, format, and mount all raw disks, including ones not attached to the disk array controller.
+
+  **Please note:** If HBA-Mode (JBOD) is selected, the following RAID option values will be ignored.
+
+9. **RAID Level** - Select your desired RAID level:
+  - RAID-10
+  - RAID-5
+  - RAID-1
+  - RAID-0
+10. **Set Encryption** - Select whether you would like to encrypt the RAID volumes.
+
+  **Please note:** The encryption options below will only be applied if you chose RAID-Mode (RAID-X) above. If it is set to HBA-Mode (JBOD), the following option values will be ignored.
+
+11. **Security Key** - Enter the desired Security Key to set for disk encryption.
+
+  **Please note:**	A password must be at least 9 characters and contain at least 3 of the following:
+  - uppercase letters
+  - lowercase letters
+  - numbers
+  - symbols
+
+12. **Key ID** - Enter the desired Key ID to set for disk encryption
+
+13. **Callbacks** - Configure your optional Callbacks:
+
+  - HTTP
+    - Status level:
+      - Debug
+      - Error
+      - Result
+    - URL:
+      - http://callback/url
+  - Email
+    - Status level:
+      - Debug
+      - Error
+      - Result
+    - Subject:
+      - Your email Subject
+    - Recipient
+      - john.doe@ctl.io
+
+
+14. Click **run**
+
+Once the job is submitted to Runner, you will be directed to the Job Execution page, where you are able to follow the progress of the job.
+
+ ![Bare Metal Job Execution](../images/bare_metal_runner_disks_5.png)
+
+When the job completes successfully, the status bar will turn green (as above).
+
+### Before and After
+
+Disk configuration (Windows Server 2012) before running the Bare Metal Storage Configuration Runner job on a Bare Metal server with 1 x 128GB SSD, 2 x 800GB SSD and 4 x 4TB 7200rpm disks.
+
+ ![Bare Metal Job Execution](../images/bare_metal_runner_disks_6.png)
+
+After running the Bare Metal Storage Configuration Runner job using the RAID-10 option.
+
+ ![Bare Metal Job Execution](../images/bare_metal_runner_disks_7.png)
+
+If you have any issues, please check out the [Runner FAQ](../runner/runner-faqs.md) in our Knowledge Base.

```

Comment:
Change link to `../Runner/runner-faqs.md`. The filenames in links are case sensitive.